test/test_app_logging.py

```diff
@@ -85,3 +93,40 @@ def fail():
                             azul_log.output[1],
                             'INFO:azul.chalice:Returning 500 response. To log headers and body, set AZUL_DEBUG to 1.'
                         )
+
+
+class TestPermittedWarnings(AzulUnitTestCase):
+
+    def test_permitted_warnings(self):
+        warnings.warn(" unclosed <ssl.SSLSocket fd=30, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM,"
```

Comment:
I think one of these would be sufficient.

Reply:
And a comment should explain why there are no assertions. That tearDownClass would raise if these one of these slipped through.